similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/DefaultPatientSimilarityView.java

```diff
@@ -368,7 +369,7 @@ public double getScore()
      *
      * @return the genotype similarity view for this pair of patients
      */
-    private GenotypeSimilarityView getGenotypeSimilarity()
+    private PatientGenotypeSimilarityView getGenotypeSimilarity()
     {
         if (this.matchedGenes == null) {
             this.matchedGenes = new RestrictedGenotypeSimilarityView(this.match, this.reference, this.access);
```

Comment:
Indeed, it shouldn't anymore. Initially there was *only* the `Restricted`
genotype view. Now that there is a `Default`, it should return that insted.

On Thu, Apr 30, 2015 at 4:30 AM, Sergiu Dumitriu <notifications@github.com>
wrote:

> In
> similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/DefaultPatientSimilarityView.java
> <https://github.com/phenotips/patient-network/pull/66#discussion_r29411345>
> :
>
> > @@ -368,7 +369,7 @@ public double getScore()
> >       *
> >       * @return the genotype similarity view for this pair of patients
> >       */
> > -    private GenotypeSimilarityView getGenotypeSimilarity()
> > +    private PatientGenotypeSimilarityView getGenotypeSimilarity()
> >      {
> >          if (this.matchedGenes == null) {
> >              this.matchedGenes = new RestrictedGenotypeSimilarityView(this.match, this.reference, this.access);
>
> I wonder why does the Default patient view return a Restricted genotype
> view...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/phenotips/patient-network/pull/66/files#r29411345>.
>